CONTRIBUTING.md

```diff
@@ -5,22 +5,22 @@ Welcome stranger!
 If you have come here to learn how to contribute to mdBook, we have some tips for you!
 
 First of all, don't hesitate to ask questions!
-Use the [issue tracker](https://github.com/azerupi/mdBook/issues), no question is too simple.
-If we don't respond in a couple of days, ping us @azerupi, @steveklabnik, @frewsxcv it might just be that we forgot. :wink:
+Use the [issue tracker](https://github.com/rust-lang-nursery/mdBook/issues), no question is too simple.
+If we don't respond in a couple of days, ping us @rust-lang-nursery, @steveklabnik, @frewsxcv it might just be that we forgot. :wink:
```

Comment:
Im not sure that pinging rust-lang-nursery will work as expected but you can add me and Michael-F-Bryan

Reply:
Feel free to chuck me at the start of the list so people will ping me first, I think I've got a bit more spare time than the other guys.